engines/aio/Dockerfile

```diff
@@ -34,15 +34,23 @@ COPY target/${env.project_artifactId}-${env.project_version}.jar /usr/bin
 #removing perl-ExtUtils-MakeMaker cascades to remove exiftools
 RUN ln /usr/bin/${env.project_artifactId}-${env.project_version}.jar /usr/bin/${env.project_artifactId}.jar && \
     yum install -y $IMAGEMAGICK_DEP_RPM_URL && \
-    yum install -y $IMAGEMAGICK_LIB_RPM_URL $IMAGEMAGICK_RPM_URL && \
-    rpm -e --nodeps libgs && \
```

Comment:
Where did this line go / why was it removed entirely:

`rpm -e --nodeps libgs`?